pages/index.js

```diff
@@ -5,6 +5,8 @@ import Filters from '../components/filters.js'
 // import Routes from '../components/routes.js'
 import { promises as fs } from 'fs';
 import path from 'path';
+import styles from 'mapbox/lib/services/styles';
```

Comment:
Unused but added in preparation for embedding mapbox maps?

Reply:
I don't think I added this, must have been either when I installed or some auto text